src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2866,7 +3092,18 @@ void processContextElement
           }
         }
 
-        processSubscriptions(enP, subsToNotify, &errReason, tenant, xauthToken, servicePathV);
+        /* Build CER used for notifying (if needed). Service Path vector shouldn't have more than
+         * one item, thus it should be safer to get item 0 */
+        ContextElementResponse* notifyCerP = new ContextElementResponse(ceP);
+        notifyCerP->contextElement.entityId.servicePath = servicePathV.size() > 0? servicePathV[0] : "";
+
+        processSubscriptions(subsToNotify, notifyCerP, &errReason, tenant, xauthToken);
+
+        // FIXME P10: I would say that the memory allocated previously for notifyCerP should
+        // be released, but uncomentaing this line produces a double-free crash. Mabye somebody
+        // has already released before?
+        //notifyCerP->release();
+        //delete notifyCerP;
```

Comment:
processOnChangeConditionForUpdateContext:

    caP = notifyCerP->contextElement.contextAttributeVector.get(ix);

It seems you point to an attribute of notifyCerP.
If you copy the attribute instead, you might be able to free  notifyCerP  where you want